packages/rsync/ChangeLog

```diff
@@ -2,3 +2,9 @@
 
 	* 3.1.1-1 :
 	Initial version
+
+2017-04-16 Fitz Agard <fhagard@lightcubesolutions.com>
+
+	* 3.1.2-1 :
+	Upgrade to 3.1.2
+
```

Comment:
What I've been doing in ChangeLog files so far is to put newer entries at the top. It also seems to be a convention used elsewhere. Would you mind just changing that around?